components/Exercises/ExerciseList/index.jsx

```diff
@@ -21,50 +14,8 @@ export default function ExerciseList() {
       <Link href="/">Workouts</Link>
       <Heading>Exercises</Heading>
       {exercises.map((exercise) => (
-        <StyledExerciseCard>
-          <Image
-            src={exercise.imageUrl || "power.svg"}
-            alt="exercise image"
-            width={50}
-            height={50}
-          ></Image>
-          <article>
-            <HeadingLarge>{exercise.name}</HeadingLarge>
-            <Heading>Musclegroups</Heading>
-            <StyledList>
-              {exercise.muscleGroups.map((muscle) => (
-                <HeadingTiny>// {muscle}</HeadingTiny>
-              ))}
-            </StyledList>
-          </article>
-        </StyledExerciseCard>
+        <ExerciseCard exercise={exercise} />
```

Comment:
```suggestion
        <ExerciseCard key={exercise._id} exercise={exercise} />
```
Let's add a key here